Framework/DataHandling/src/ISISRunLogs.cpp

```diff
@@ -99,5 +132,28 @@ void ISISRunLogs::addPeriodLog(const int period, API::Run &exptRun) {
   exptRun.addLogData(m_logParser->createPeriodLog(period));
 }
 
+std::vector<std::string>
+ISISRunLogs::getLogNamesExcludedFromFiltering(const API::Run &run) {
+  std::vector<std::string> retVal;
+  if (run.hasProperty(LogParser::statusLogName())) {
+    retVal.emplace_back(LogParser::statusLogName());
+  }
+  if (run.hasProperty(LogParser::periodsLogName())) {
+    retVal.emplace_back(LogParser::periodsLogName());
+  }
+  const auto props = run.getProperties();
```

Comment:
Although the values are pointers I think a reference is needed here to avoid copying the container?

```suggestion
  const auto &props = run.getProperties();
```